Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -249,14 +249,24 @@
   categories:
     - UplinkUtility
 
-#TODO: Increase the price of this to 4-5/remove it when we get encrpytion keys
+#TODO: Increase the price to 4 when flashbang prof.
```

Comment:
why should this be flashbang proof? The only thing that should stop flashes are sunglasses.

Reply:
Because original do, someway, don't know, i was not security player.
This was writen in description of radio, i think it is migrated from original